inventory/forms.py

```diff
@@ -157,14 +158,8 @@ class Meta:
     def __init__(self, *args, **kwargs):
         user = kwargs.pop('user')
         super(ItemForm, self).__init__(*args, **kwargs)
-        self.fields["uom"].queryset = Uom.objects.filter(company=user.company)
-        for field in self.fields:
-            if field == 'description':
-                self.fields[field].widget.attrs['class'] = 'form-control'
-            elif self.fields[field].widget.input_type == 'select':
-                self.fields[field].widget.attrs['class'] = 'form-control  custom-select'
-            else:
-                self.fields[field].widget.attrs['class'] = 'form-control'
+        self.fields['uom'].queryset = Uom.objects.all()
```

Comment:
we need to filter the query set to return all what is related to my current company only